index.html

```diff
@@ -863,15 +863,17 @@ <h3>
       [=digital credential=].
     </p>
     <p>
-      [=User mediation=] is always
-      {{CredentialMediationRequirement/"required"}}. [=Request a
-      credential|Requesting a DigitalCredential credential=] does not support
-      {{CredentialMediationRequirement/"conditional"}},
-      {{CredentialMediationRequirement/"optional"}}, or
-      {{CredentialMediationRequirement/"silent"}} [=user mediation=]. If
-      {{CredentialsContainer/get()}} is called with anything other than
-      {{CredentialMediationRequirement/"required"}}, a {{TypeError}} will be
-      thrown.
+      The {{DigitalCredential}} interface mandates [=user mediation=] for all
+      operations to ensure user control and consent.
+    </p>
+    <p>
+      To simplify the developer experience, for {{CredentialsContainer/get()}}
```

Comment:
So instead of talking about create and get in the same paragraph, it would be best to split this into two paragraphs: one deals with get() other deals with create()... and then link CredentialCreationOptions and CredentialRequestOptions as needed.